commit 3c78318b0764d892950f11ec8f460719d4eff6a7
Author:     Dongkyun Son <dongkyun.s@samsung.com>
AuthorDate: Mon Oct 26 16:19:54 2026 +0900
Commit:     Dongkyun Son <dongkyun.s@samsung.com>
CommitDate: Wed Oct 28 12:28:41 2026 +0900

    Tizen-Base-Toolchain, Tizen-Base-Dev: update meta and snapshot
    
    This is to enable build along with Tizen-Base rpms one time temporarily.
    
    Change-Id: Ide23bdad77985a609626ab3fea253800f8548f29
    Signed-off-by: Dongkyun Son <dongkyun.s@samsung.com>
---
 TIZEN/Tizen/Tizen-Base-Dev/project_meta        | 2 +-
 TIZEN/Tizen/Tizen-Base-Dev/snapshot_info       | 6 +++---
 TIZEN/Tizen/Tizen-Base-Toolchain/project_meta  | 2 +-
 TIZEN/Tizen/Tizen-Base-Toolchain/snapshot_info | 6 +++---
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/TIZEN/Tizen/Tizen-Base-Dev/project_meta b/TIZEN/Tizen/Tizen-Base-Dev/project_meta
index 774c1b3..12d2f34 100644
--- a/TIZEN/Tizen/Tizen-Base-Dev/project_meta
+++ b/TIZEN/Tizen/Tizen-Base-Dev/project_meta
@@ -3,7 +3,7 @@
     <enable/>
   </debuginfo>
   <repository name="standard">
-    <path project="DOD:Tizen-Base-Dev" repository="standard" />
+    <path project="DOD:Tizen-Base" repository="standard" />
     <arch name="armv7l"/>
     <arch name="aarch64"/>
     <arch name="x86_64"/>
diff --git a/TIZEN/Tizen/Tizen-Base-Dev/snapshot_info b/TIZEN/Tizen/Tizen-Base-Dev/snapshot_info
index 28d7e98..c407497 100644
--- a/TIZEN/Tizen/Tizen-Base-Dev/snapshot_info
+++ b/TIZEN/Tizen/Tizen-Base-Dev/snapshot_info
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <latest target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
-  <reference target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/"/>
-  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/">
+  <latest target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/"/>
+  <reference target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/"/>
+  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/">
     <repository name="standard">
       <type name="standard"/>
     </repository>
diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/project_meta b/TIZEN/Tizen/Tizen-Base-Toolchain/project_meta
index 2dcffed..7fed9ad 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/project_meta
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/project_meta
@@ -3,7 +3,7 @@
     <enable/>
   </debuginfo>
   <repository name="standard">
-    <path project="DOD:Tizen-Base-Toolchain" repository="standard" />
+    <path project="DOD:Tizen-Base" repository="standard" />
     <arch name="armv7l"/>
     <arch name="aarch64"/>
     <arch name="x86_64"/>
diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/snapshot_info b/TIZEN/Tizen/Tizen-Base-Toolchain/snapshot_info
index 6ad9e4a..c407497 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/snapshot_info
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/snapshot_info
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <latest target="tizen-base-toolchain_20261026.051553" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/"/>
-  <reference target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/"/>
-  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/">
+  <latest target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/"/>
+  <reference target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/"/>
+  <base target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/">
     <repository name="standard">
       <type name="standard"/>
     </repository>
